students/MiguelNeto/index.html

```diff
@@ -2,24 +2,81 @@
 <html lang="en">
 
 <head>
-  <title>Hello, world!</title>
+  <title>Miguel Neto Page</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <meta property="og:title" content="Miguel Neto Page">
+  <meta property="og:property" content="profile">
+  <meta property="og:url" content="http://www.miguelneto.com/">
+  <meta property="og:image" content="https://nit.pt/wp-content/uploads/2018/08/fd5014cab115b944242b07ccc5d03c74-754x394.jpg">
+  <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700" rel="stylesheet">
   <link rel="stylesheet" href="style.css">
 </head>
 
-<body>
-  <h1>Hello World</h1>
-    <nav>
-        <ul>
-            <li class="nav-button"><a href="#" >Home!</a></li>
-            <li class="nav-button"><a href="#" >About me</a></li>
-            <li class="nav-button"><a href="#" >Contact</a></li>
+<body itemscope itemtype="http://schema.org/Person">
+    <header>
+        <div class="flex">
+            <a href="#"><img src="home.png" alt="home" class="home-icon"></a>
+            <nav>
+                <ul class="list-nav">
+                    <il class="list-nav-item"><a href="#Areas-Interest">Areas Interest</a></il>
+                    <il class="list-nav-item"><a href="#about-me">About Me</a></il>
+                    <il class="list-nav-item"><a href="#Contacts">Contacts</a></il>
+                </ul>
+            </nav>
+            
+        </div>
+    </header>
```

Comment:
Careful I think that you're using 4 spaces for indentation, best practice is use 2 spaces indentation. And try not to leave spaces (enters) if you don't really need them